index.html

```diff
@@ -694,18 +694,15 @@
 In general, providing [=consent=] should be rare, intentional, and temporary.
 
 When an [=opt-out=] mechanism exists, it should preferably work with a
-<dfn>global opt-out</dfn> mechanism. The function of a [=global opt-out=] mechanism is to
-rectify the <dfn>automation asymmetry</dfn> whereby service providers can automate
-[=data processing=] but [=people=] have to take manual action to prevent it. A good example of a
-[=global opt-out=] mechanism is the <em>Global Privacy Control</em> [[?GPC]].
-
-Conceptually, a [=global opt-out=] mechanism is an automaton operating as part of the
-[=user agent=]. It is equivalent to a robot that would carry out a [=person=]'s instructions
-by pressing an [=opt-out=] button (or a similar expression of the [=person=]'s rights) with every
-interaction that the [=person=] has with a site. (For instance, the [=person=] may be objecting to [=processing=]
-based on legitimate interest, withdrawing [=consent=] to specific [=purposes=], or requesting that
-their data not be sold or shared.) The [=user=] is effectively delegating the expression
-of their [=opt-out=] to their [=user agent=], which helps rectify [=automation asymmetry=].
+<dfn>global opt-out</dfn> mechanism. Conceptually, a [=global opt-out=] mechanism is an 
+automaton operating as part of the [=user agent=]. It is equivalent to a robot that would carry 
+out a [=person=]'s instructions by pressing an [=opt-out=] button (or a similar expression of 
+the [=person=]'s rights) with every interaction that the [=person=] has with a site. (For 
+instance, the [=person=] may be objecting to [=processing=] based on legitimate interest, 
+withdrawing [=consent=] to specific [=purposes=], or requesting that their data not be sold or 
+shared.) The [=user=] is effectively delegating the expression of their [=opt-out=] to their 
+[=user agent=], which helps rectify [=automation asymmetry=]. A good example of a [=global opt-out=] 
+mechanism is the <em>Global Privacy Control</em> [[?GPC]].
```

Comment:
This might flow a little better as:

```suggestion
[=user agent=], which helps rectify [=automation asymmetry=]. The <em>Global Privacy
Control</em> [[?GPC]] is a good example of a [=global opt-out=] mechanism.
```